manifests/c/CloudFoundry/CLI/v8/8.7.1/CloudFoundry.CLI.v8.installer.yaml

```diff
@@ -0,0 +1,49 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=QUSU.CRLF.7-4-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CloudFoundry.CLI.v8
+PackageVersion: 8.7.1
+InstallerType: zip
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: cf8.exe
+    PortableCommandAlias: cf8
+  - RelativeFilePath: cf.exe
+    PortableCommandAlias: cf
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli_8.7.1_winx64.zip
+  InstallerSha256: 4E61629DFDCDF19FAF5F81E1613904816363CEE313D0BE4CA6CA2F054E83B029
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: cf.exe
+    PortableCommandAlias: cf
+  - RelativeFilePath: cf8.exe
+    PortableCommandAlias: cf8
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli_8.7.1_win32.zip
+  InstallerSha256: C4AD867E43A6F5E8F0C4FDEE77B1B7036E23BED455BCFF268E1811EA414E33AC
+- Architecture: x64
+  NestedInstallerType: inno
+  NestedInstallerFiles:
+  - RelativeFilePath: cf8_installer.exe
+  Scope: machine
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli-installer_8.7.1_winx64.zip
+  InstallerSha256: 2B15E48A8B56E63B911ADEF36B5A54D401901B23B6636539BF1544B11E465DE2
+  AppsAndFeaturesEntries:
+  - DisplayVersion: v8.7.1
+    ProductCode: Cloud Foundry CLI_is1
+  ElevationRequirement: elevatesSelf
+- Architecture: x86
+  NestedInstallerType: inno
+  NestedInstallerFiles:
+  - RelativeFilePath: cf8_installer.exe
+  Scope: machine
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli-installer_8.7.1_win32.zip
+  InstallerSha256: D408F21753E2CB09D558DA832BC9D84F23DD1D081B4437D21BB42B3E8A782092
+  AppsAndFeaturesEntries:
+  - DisplayVersion: v8.7.1
+    ProductCode: Cloud Foundry CLI_is1
+  ElevationRequirement: elevatesSelf
```

Comment:
```suggestion
```

DisplayVersion is not `v8.7.1` in this case. DisplayVersion must match whatever is written to Control Panel / Apps&Features after installing the package. In this case, the `inno` installers don't write a proper version. See -

![image](https://github.com/microsoft/winget-pkgs/assets/88161975/2ceaefd1-9fae-40f0-9d70-8594bb56fe25)

Now we can modify the installer nodes and set DisplayVersion to `VERSION`, but there's another issue with this approach. All versions of this package seem to write the same `VERSION` to Control Panel. WinGet requires the DisplayVersion to be unique for every version of a package. This unfortunately means that we cannot offer the `inno` installers for this package. At the very least, we could only offer it in the latest version and go back and remove in the rest so that it becomes the 'unique' version. But that also has problems of its own (users can face an "infinite upgrade" issue and puts more burden on the contributor to know the context around the package). We should probably just remove these installer nodes as I've suggested above, and only offer the "portable" versions.

Reply:
I initially tried writing `VERSION` to it, but just like you have mentioned, it resulted in violating the unique property. So, just as suggested, I will remove the `inno` installation for this package.